pkg/termios/sgtty.go

```diff
@@ -4,21 +4,11 @@
 
 package termios
 
-import (
-	"fmt"
-	"log"
-	"reflect"
-	"sort"
-	"strconv"
-
-	"golang.org/x/sys/unix"
-)
-
 type (
-	// tty is an os-independent version of the combined info in termios and window size structs.
+	// TTY is an os-independent version of the combined info in termios and window size structs.
 	// It is used to get/set info to the termios functions as well as marshal/unmarshal data
-	// in JSON formwt for dump and loading.
-	tty struct {
+	// in JSON format for dump and loading.
+	TTY struct {
```

Comment:
i'd suggest merging this last struct into termios.go

TTYIO is the same for linux and plan9, so i'd put that there, too (especially since termios.go operates on TTYIO)

Reply:
on plan 9, there will be a bit more to TTY as I add in the capabilities, hence the separation. on Plan9, data and ctl are different files, with different names.

Reply:
sgtty.go had no reason to live, it's gone :-)